main.js

```diff
@@ -1,115 +1,120 @@
-const electron = require('electron');
-// Module to control application life.
-const app = electron.app;
-// Module to create native browser window.
-const BrowserWindow = electron.BrowserWindow;
+const {app, BrowserWindow, dialog} = require('electron');
 
 const path = require('path');
 const url = require('url');
 const fs = require('fs');
+const winston = require('winston');
+
+let level = 'warn';
+if (process.env.DEBUG && process.env.DEBUG.includes('hatch-previewer'))
+    level = 'debug';
```

Comment:
Maybe wrap the case in parens or space the next line one more emty newline down? It's a bit confusing to read given it's visual context.